### PR TITLE
Test: visual testing integration

### LINKED ISSUE #115: Analyze Visual Regression Testing Libraries and Conduct a POC
Analyze and evaluate available Visual Regression Testing libraries to determine their suitability for our project. Perform a POC with the selected library to validate its capabilities and effectiveness in detecting visual changes in our application.

### PR DESCRIPTION
**Pull request by ramzes-84**
## What type of PR is this? (select all that apply)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [x] 📝 Documentation Update

## Description

Basic Playwright integration to evaluate its usability.
Please meet [Readme file](https://github.com/rolling-scopes/site/blob/test/115-visual-testing/readme/end-to-end-testing.md).
Don't forget `npm install`.

## Related Tickets & Documents

- Related Issue #115 
- Closes #115 

## Screenshots, Recordings

origin-diff

## Added/updated tests?

- [x] 👌 Yes
- [ ] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

### CODE REVIEW

**Review by andron13 — COMMENTED** [review]
I see no reason to create a separate folder for the file as here: `readme/end-to-end-testing/end-to-end-testing.md`, so far we have been writing in a common folder.

File: readme/assets/UI.jpg
[the remark this answers is not in the document]
Comment (ramzes-84) [reply readme/assets/UI.jpg:1]: I made it at the beginning (just like you did in your `image-optimization` PR), but @andron13 recommended me not to do this.

File: visualTesting/screenshots.spec.ts
[the remark this answers is not in the document]
Comment (ramzes-84) [reply visualTesting/screenshots.spec.ts:1]: Sure

File: playwright.config.ts
[the remark this answers is not in the document]
Comment (ramzes-84) [reply playwright.config.ts:1]: Removed

File: readme/assets/UI.jpg
[the remark this answers is not in the document]
Comment (ramzes-84) [reply readme/assets/UI.jpg:1]: Denied

File: visualTesting/screenshots.spec.ts
[the remark this answers is not in the document]
Comment (ramzes-84) [reply visualTesting/screenshots.spec.ts:1]: Replaced

**Review by Quiddlee — APPROVED** [review]

**Pull request merged**